### DISCUSSION TITLE
Can't use on meeting app like Zoom/Google Meet (macOS Monterey 12)

### DISCUSSION THREAD

**Discussion by sassa7777** [ptrumpis/snap-camera-server#78] [Q&A] [answered]
I cant find snap camera option in zoom and google meet

**Comment by vitorviesi** [comment]
Same here. Any solutions?

**Comment by sassa7777** [comment]
> Please see here [#76 (comment)](https://github.com/ptrumpis/snap-camera-server/discussions/76#discussioncomment-7470839)

thanks for reply, but Im having the issue on monterey

**Reply by ptrumpis** [reply]
Ok, I changed the title.
I can't help you with the issue but another macOS user might know the answer.

**Comment by ptrumpis** [comment, marked as answer]
If these guides do not help:
- [Control access to the camera on macOS Monterey 12](https://support.apple.com/guide/mac-help/mchlf6d108da/12.0/mac/12.0) 
- [Control access to the camera on macOS Ventura 13](https://support.apple.com/guide/mac-help/mchlf6d108da/13.0/mac/13.0)
- [Snap Camera not appearing as a source to other apps on MacOS Sonoma](https://github.com/ptrumpis/snap-camera-server/discussions/76#discussioncomment-7472043)

You can try to fix Zoom with these steps:
1. remove signing
```
sudo codesign --remove-signature "/Applications/Zoom.app"
```

2. re-sign
```
sudo codesign --force --deep --sign - "/Applications/Zoom.app"
```

For any other application you need to replace `"/Applications/Zoom.app"` with your application path.

For **Discord** this would be:
`sudo codesign --remove-signature "/Applications/Discord.app"`
`sudo codesign --force --deep --sign - "/Applications/Discord.app"`

Installing and patching Snap Camera prior to any other application can also help.
